build.gradle

```diff
@@ -99,7 +102,7 @@ allprojects {
     //   * any new checkers have been added,
     //   * the patch level is 9 (keep the patch level as a single digit), or
     //   * backward-incompatible changes have been made to APIs or elsewhere.
-    version '3.2.0'
+    version '3.2.1'
```

Comment:
Why is the release number already incremented?  Maybe it's a temporary artifact of the lack of SNAPSHOT version numbers.